packages/ckeditor5-typing/src/deleteobserver.js

```diff
@@ -211,3 +215,73 @@ export default class DeleteObserver extends Observer {
  * current selection should be used.
  * @param {String} data.inputType The `beforeinput` event type that caused the deletion.
  */
+
+// Enables workaround for the issue https://github.com/ckeditor/ckeditor5/issues/11904.
+function enableChromeWorkaround( observer ) {
+	const view = observer.view;
+	const document = view.document;
+
+	let pressedKeyCode = null;
+	let beforeInputReceived = false;
+
+	document.on( 'keydown', ( evt, { keyCode } ) => {
+		pressedKeyCode = keyCode;
+		beforeInputReceived = false;
+	} );
+
+	document.on( 'keyup', ( evt, { keyCode, domEvent } ) => {
+		const selection = document.selection;
+		const shouldFireDeleteEvent = observer.isEnabled &&
+			keyCode == pressedKeyCode &&
+			isDeleteKeyCode( keyCode ) &&
+			!selection.isCollapsed &&
+			!beforeInputReceived;
+
+		pressedKeyCode = null;
+
+		if ( shouldFireDeleteEvent ) {
+			const targetRange = selection.getFirstRange();
+			const eventInfo = new BubblingEventInfo( document, 'delete', targetRange );
+			const deleteData = {
+				unit: DELETE_SELECTION,
+				direction: getDeleteDirection( keyCode ),
+				selectionToRemove: selection
+			};
+
+			document.fire( eventInfo, new DomEventData( document, domEvent, deleteData ) );
+
+			if ( eventInfo.stop.called ) {
```

Comment:
Why is this needed? We use that event just to detect missing event so maybe we should not allow to stop the original `keyup` event?